.travis.yml

```diff
@@ -42,6 +42,7 @@ before_script:
 - docker build --no-cache -t masschallenge/front-end ../front-end
 - docker run -v $(pwd)/../front-end/dist:/usr/src/app/dist -t masschallenge/front-end
 - cp -r ../front-end/dist web/impact/static/front-end-dist
+- cp -r ../front-end/dist web/impact/static-compiled/front-end-dist
```

Comment:
what does this change do?

Reply:
impact API server serves it's assets from the `static-compiled` directory. Without this, my new route was not being recognised.